modular/gater/request_context.go

```diff
@@ -62,6 +76,10 @@ func NewRequestContext(r *http.Request, g *GateModular) (*RequestContext, error)
 		vars:       vars,
 		startTime:  time.Now(),
 	}
+	log.Infof("routerName is %s", routerName)
+	if slices.Contains(skipAuthRouterNames, routerName) {
```

Comment:
why can't they be exactly the same, but have to be `contains`?

Reply:
not an issue, as discussed.